### ISSUE TITLE
Delete Statement with Inner Select (e.g. delete duplicates with one statement)

### ISSUE THREAD

**Issue by loms0815** [crate/crate#3471]
Hi, 
is it possible to do someting like this:

delete from table test where id in
(
    select id from test2
)
##

**Comment by joemoe** [comment]
hi @loms0815, no we are not able to do that as we don't support subselects at the moment. they are already on our backlog. i'll keep you posted.

**Comment by mfussenegger** [comment]
We've added support scalar subqueries (+ IN & ANY) in delete statements  in master - it will become available with the next feature release (2.3)

So something like this will work:

```
cr8 run-crate latest-nightly -- @crash <<EOF
    create table t1 (x int);
    create table t2 (x int);
    insert into t1 (x) values (1), (2), (3);
    insert into t2 (x) values (1), (2);
    refresh table t1, t2;
    delete from t1 where x in (select x from t2);
    refresh table t1;
    select * from t1;
EOF
```
